frontends/infinite-corridor/src/pages/NotFoundPage.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import Container from "@mui/material/Container"
+import Card from "@mui/material/Card"
+import CardContent from "@mui/material/CardContent"
+import CardActions from "@mui/material/CardActions"
+import Button from "@mui/material/Button"
+
+const NotFoundPage: React.FC = () => {
+  return (
+    <Container maxWidth="sm">
+      <Card sx={{ marginTop: "1rem" }}>
+        <CardContent>
+          <h1>404 Not Found Error</h1>
+          Resource Not Found
+        </CardContent>
+        <CardActions>
+          <Button variant="outlined" href="/infinite">
```

Comment:
I think there's a `HOME` const in `urls.ts`.

Reply:
Added to both